index.html

```diff
@@ -93,7 +93,7 @@
             <h2 class="section-heading text-uppercase">See it in action now</h2>
           </div>
         </div>
-        <div class="embed-responsive embed-responsive-16by9">
+        <div class="embed-responsive embed-responsive-16by9 about_video">
```

Comment:
the other classes use `-` instead of `_`, could you please change to `about-video` to be consistent